addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/MongoDbInjectionScanRule.java

```diff
@@ -70,7 +70,7 @@ public class MongoDbInjectionScanRule extends AbstractAppParamPlugin {
     private static final int BLIND_REQUEST_LIMIT = 5;
     // time-based attack detection will try to take less than the following number of seconds
     // note: detection of this length will generally only happen in the positive (detecting) case.
-    private static final double BLIND_SECONDS_LIMIT = 20.0;
+    private double BLIND_SECONDS_LIMIT;
```

Comment:
If it's no longer a constant it shouldn't be full caps and it should be moved down to 114-124 so that it's grouped with similar vars.